job_templates/packaging_job.xml.em

```diff
@@ -134,7 +134,7 @@ if [ -n "${CI_AMENT_TEST_ARGS+x}" ]; then
   if [[ "$CI_AMENT_TEST_ARGS" != *-- ]]; then
     CI_AMENT_TEST_ARGS="$CI_AMENT_TEST_ARGS --"
   fi
-  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS"
+  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS --"
```

Comment:
Can you explain why this was necessary ? I would have expected line134 to take care of adding the delimiters ? I think this will break if you pass test argument that already have the `--` delimiter.
Maybe the condition is too strict and should check if the AMENT_TEST_ARGS ends-with `--` or `-- `?